.github/workflows/check.yaml

```diff
@@ -39,4 +39,8 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
-          args: --issues-exit-code=0 -E gofmt
+          # The suppression of rule `errcheck` may be removed after adding
```

Comment:
Please, also add a link and a comment here about why we didn't set nolints in code.

https://github.com/tarantool/go-tarantool/pull/160#discussion_r858608221

Reply:
Added.